scripts/builder/checker_builder/templates/constants.ml.jinja

```diff
@@ -0,0 +1,7 @@
+(* Automatically generated by checker_builder. DO NOT EDIT DIRECTLY! *)
+
+open Common
+open FixedPoint
+open Tok
+
+(* TODO: Implement this template *)
```

Comment:
I think this is not needed yet, right? In a separate PR we can generate the `constants.ml` file as well.